Source/RageCommunity.Library/Wrappers/NativePadWrappers.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
```

Comment:
filename: pad or ped wrappers?

Reply:
Pad and Ped are two separate namespaces.